cli/Valet/PhpFpm.php

```diff
@@ -105,13 +108,18 @@ function stop()
      */
     function fpmConfigPath()
     {
-        $confLookup = [
-            'php' => '/usr/local/etc/php/7.2/php-fpm.d/www.conf',
-            'php72' => '/usr/local/etc/php/7.2/php-fpm.d/www.conf',
-            'php71' => '/usr/local/etc/php/7.1/php-fpm.d/www.conf',
-            'php70' => '/usr/local/etc/php/7.0/php-fpm.d/www.conf',
-            'php56' => '/usr/local/etc/php/5.6/php-fpm.conf',
-        ];
+        $confLookup = [];
+
+        foreach (Brew::SUPPORTED_PHP_VERSIONS as $version) {
+            $versionNormalized = preg_replace('/([^\d\.])/', '', $version);
+            $versionNormalized = $versionNormalized === ''
+                ? '7.2'
+                : $versionNormalized;
+
+            $confLookup[$version] = $versionNormalized === '5.6'
+                ? '/usr/local/etc/php/5.6/php-fpm.conf'
+                : "/usr/local/etc/php/${versionNormalized}/php-fpm.d/www.conf";
+        }
```

Comment:
I would prefer to keep the explicit lookup table instead of trying to get too clever and dynamic here unless there's a *very* good reason not to?

Reply:
@adamwathan  the only exception is `php @ 5.6`. I believe it is explicit enough and adding a new version of PHP implies modifying less code in fewer places.